src/scss/themes/inverted.scss

```diff
@@ -0,0 +1,3 @@
+#GUIContainer[data-theme="inverted"] {
+  filter: invert(100%);
```

Comment:
Is this actually how this should work? I'm not too familiar with this type of accessibility work so not sure if there is more that needs to be done? Thoughts?

Reply:
This is a first-pass at a dark theme -- it works, but there are way better mechanisms for theming. Using `data-theme='theme-name'` is a good approach (well, it's an approach used quite often), but a better theme would have scss inside this outer selector to apply predetermined colors and fonts (for a larger-font variant).

The theming part is going to take some design -- this was the simplest dark theme I could come up with to test the process and see if you'd even be interested in a PR -- I didn't want to spend ages on polishing a theme only to have the PR either sit or just be plain rejected.

Reply:
When the time comes to design proper themes, there should be at least (imo)

- a beautiful dark theme that most people wanting a dark theme would use (like how Dark Reader makes the web look)
- a high contrast dark theme for accessibility (with larger fonts)
- a high contrast light theme for accessibility (with larger fonts)
  - for some people, light-on-dark is difficult to deal with -- they just need larger fonts and inputs

both of the latter should be tested on machines where default font sizes have been set larger, to see if those settings apply to the addon's overlay. I have a friend who can test for me there. I can also get feedback from him on accessible color choices -- I remember having a discussion with him quite a while ago about the accessibility themes in Windows and what did and didn't work there.

Reply:
Sounds good, looking forward to seeing more!